src/App.js

```diff
@@ -78,15 +81,44 @@ const routes = [
   },
 ];
 
-const Base = props => (
-  <div className={css.App}>
-    <Nav {...props} />
-    {props.children}
-  </div>
-);
+class Base extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      registerModal: false,
+      signInModal: false,
+    };
+  }
+  handleModalToggle = modalName => {
+    const modal =
```

Comment:
```suggestion
    const otherModal =
```
Maybe name this `otherModal` so it's a little more clear.

Reply:
thanks for the suggestion